package/MDAnalysis/analysis/base.py

```diff
@@ -90,13 +90,76 @@
    plt.ylabel("RMSD (Å)")
 
 
+Using parallelization in your analysis
+-------------------------------------
+--------------------------------------
+Starting from 2.8.0, MDAnalysis ``AnalysisBase`` subclasses can run on a backend
+that supports parallelization (see :mod:`MDAnalysis.analysis.backends`). By
+default, only one backend is supported -- built-in `multiprocessing`, that
+processes parts of a trajectory running separate *processes*, i.e. utilizing
+multi-core processors properly. For now, parallelization is added to
+:class:`MDAnalysis.analysis.RMS.RMSD`, but by 3.0 version it will be introduced
+to all subclasses that can support it.
+
+In order to use that feature, simply add ``backend='multiprocessing'`` to your
+run, and supply it with proper ``n_workers`` (use ``multiprocessing.cpu_count()``
+for the maximum available on your machine):
+
+.. code-block:: python
+
+   import multiprocessing
+   import MDAnalysis as mda
+   from MDAnalysisTests.datafiles import PSF, DCD
+   from MDAnalysis.analysis.rms import RMSD
+   from MDAnalysis.analysis.align import AverageStructure
+
+   # initialize the universe
+   u = mda.Universe(PSF, DCD)
+
+   # calculate average structure for reference
+   avg = AverageStructure(mobile=u).run()
+   ref = avg.results.universe
+
+   # initialize RMSD run
+   rmsd = RMSD(u, ref, select='backbone')
+   rmsd.run(backend='multiprocessing', n_workers=multiprocessing.cpu_count())
+
+For now, you have to be verbal and specify both ``backend`` and ``n_workers``,
+since the feature is new and there are no good defaults for it. For example,
+if you specify too big `n_workers`, and your trajectory frames are big,
+you might get and out-of-memory error when executing your run.
```

Comment:
what happens if someone passes `n_workers` to the `run` method of their analysis, and passes an instantiated backend that has `n_workers` set?
i.e.
```python
from MDAnalysis.analysis.backends import BackendMultiprocessing

backend = BackendMultiprocessing(n_workers=10)
my_analysis.run(n_workers=2, backend=backend)
```
(This probably isn't something users would do with the builtin backends, but they might with their own backends, e.g. if they have a backend for using Dask Distributed they will need to pass an instance rather than then class itself)

Would 10 workers be used or 2? Either way, it might be confusing for users.

One way around this would be to remove the `n_workers` argument from `run`, and have the backends default to using 1 worker per cpu. Then if a user wants to change this they can instantiate the backend and pass this object in.

Or perhaps for now just add a note to the docstring explaining what would happen in this situation.

Reply:
(This probably isn't something users would do with the builtin backends, but they might with their own backends, e.g. if they have a backend for using Dask Distributed they will need to pass an instance rather than then class itself)